src/serializers/decoders/current/v19/decode_graph_schema.c

```
@@ -0,0 +1,324 @@
+	 * N * stopword
+	 * #properties - M
+	 * M * property: {options} */
+
+	Index idx        = NULL;
+	char *language   = SerializerIO_ReadBuffer(rdb, NULL);
+	char **stopwords = NULL;
+	
+	uint stopwords_count = SerializerIO_ReadUnsigned(rdb);
+	if(stopwords_count > 0) {
+		stopwords = arr_new(char *, stopwords_count);
+		for (uint i = 0; i < stopwords_count; i++) {
```

Cppcheck (2.20.0)

[error] Common realloc mistake

(memleakOnRealloc)